articles/cloud-storage/cs-how-python-boto-ecs.md

```diff
@@ -0,0 +1,160 @@
+---
+title: How to interact with ECS using Python and Boto| UKCloud Ltd
+description: How to use Python and Boto with ECS
+services: cloud-storage
+author: Paul Brown
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: How to use Python and Boto with ECS
+toc_fullpath: How To/cs-how-python-boto-ecs.md
```

Comment:
Change file name to cs-how-use-ecs-python-boto.md in line with file naming conventions

Don't forget to also change the actual file name!